homeassistant/components/hive/light.py

```diff
@@ -46,59 +48,56 @@ def device_info(self):
     @property
     def name(self):
         """Return the display name of this light."""
-        return self.node_name
+        return self.device["haName"]
+
+    @property
+    def available(self):
+        """Return if the device is available."""
+        return self.device["deviceData"]["online"]
 
     @property
     def device_state_attributes(self):
         """Show Device Attributes."""
-        return self.attributes
+        return {
+            CONF_AVAILABLE: self.attributes[CONF_AVAILABLE],
+            CONF_MODE: self.attributes[CONF_MODE],
+        }
 
     @property
     def brightness(self):
         """Brightness of the light (an integer in the range 1-255)."""
-        return self.session.light.get_brightness(self.node_id)
+        return self.device["status"]["brightness"]
 
     @property
     def min_mireds(self):
         """Return the coldest color_temp that this light supports."""
-        if (
-            self.light_device_type == "tuneablelight"
-            or self.light_device_type == "colourtuneablelight"
-        ):
-            return self.session.light.get_min_color_temp(self.node_id)
+        return self.device.get("min_mireds", None)
```

Comment:
None is the default value returned if the key is missing in the dict.
```suggestion
        return self.device.get("min_mireds")
```

Reply:
Fixed in latest commit